### ISSUE TITLE
Routing issue when using as plugin icon

### ISSUE THREAD

**Issue by dagwieers** [emilsvennesson/script.module.inputstreamhelper#122]
Due to an issue in **script.module.routing** we get a crash when the program icon is clicked on. This was first reported in #91 and #108 

```log
2019-08-28 20:26:38.458 T:10156   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <class 'IndexError'>
                                            Error Contents: list index out of range
                                            Traceback (most recent call last):
                                              File "C:\Users\test\AppData\Roaming\Kodi\addons\script.module.inputstreamhelper\addon.py", line 9, in <module>
                                                plugin = Plugin()
                                              File "C:\Users\test\AppData\Roaming\Kodi\addons\script.module.routing\lib\routing.py", line 57, in __init__
                                                self.handle = int(sys.argv[1]) if sys.argv[1].isdigit() else -1
                                            IndexError: list index out of range
                                            -->End of Python script error report<--
```

**Comment by dagwieers** [comment]
We reported and fixed this issue in: tamland/kodi-plugin-routing#16

**Comment by dagwieers** [comment]
A work-around was implemented in: #112, we are waiting for the fix from tamland/kodi-plugin-routing#16 to be released.

**Comment by dagwieers** [comment]
When this ever gets released, we have to remove the workaround from #112.

**Comment by dagwieers** [comment]
Routing v0.2.2 was released.
